nginx/etc/nginx/nginx-ssl.conf.ctmpl

```diff
@@ -0,0 +1,95 @@
+user  nginx;
+worker_processes  1;
```

Comment:
I personally felt it was easier to reason about and follow by making it two separate config files, simply because of the differences. By combining them, there's a lot of conditionals that increase cognitive overhead when trying to reason about what the file will look like after it's parsed. This is personal preference and I'll gladly change it (here and in the Nginx blueprint) if others feel strongly about this.

In order to determine if SSL is ready, I need to test whether two files exist on disk. I do not believe that `consul-template` provides a way of doing this natively, so I would do the test in the Nginx `reload.sh` script and pass an environment variable to the `consul-template` call.